doc-surrealdb_versioned_docs/version-1.x/surrealql/statements/update.mdx

```diff
@@ -47,15 +47,35 @@ UPDATE person:100 SET name = 'Tobie', company = 'SurrealDB', skills = ['Rust', '
 -- Update or create a record with a specific string id
 UPDATE person:tobie SET name = 'Tobie', company = 'SurrealDB', skills = ['Rust', 'Go', 'JavaScript'];
 
--- Update just a single record
```

Comment:
@Dhghomon any reason to remove this? It reads accurately.

Reply:
Ah yes, I forgot to mention this change. When I first encountered SurrealDB I read this line as meaning "you need the ONLY keyword to update a single record" when it actually has to do with what is returned (a single object instead of an array). So I think it could be removed to ensure the reader knows exactly what the difference in behaviour is when ONLY is appended.